src/Symfony/Component/Form/Extension/Core/DataMapper/PropertyPathMapper.php

```diff
@@ -73,16 +73,17 @@ public function mapFormsToData($forms, &$data)
             // Write-back is disabled if the form is not synchronized (transformation failed),
             // if the form was not submitted and if the form is disabled (modification not allowed)
             if (null !== $propertyPath && $config->getMapped() && $form->isSubmitted() && $form->isSynchronized() && !$form->isDisabled()) {
+                $propertyValue = $form->getData();
                 // If the field is of type DateTime and the data is the same skip the update to
```

Comment:
DateTime or DateTimeInterface

Please adjust the comment, thank you

Reply:
Done, thanks!